apps/docsite/sidebars.ts

```diff
@@ -58,6 +58,16 @@ const sidebars: SidebarsConfig = {
     },
     "Testing Unpublished Library Version",
     "BundleSize",
+    {
+      type:'category',
+      label: 'RFCs',
+      items: [
+        {
+          type: 'autogenerated',
+          dirName: 'rfcs'
+        }
+      ]
+    },
```

Comment:
I guess the spacing fix flow png is not needed now

Reply:
removed it